packages/@okta/vuepress-site/code/react/okta_react_sign-in_widget/index.md

```diff
@@ -15,6 +15,8 @@ This guide will walk you through integrating authentication into a React app wit
 6. [Connect the Routes](#connect-the-routes)
 7. [Start Your App](#start-your-app)
 
+> This guide was last tested with @okta/okta-signin-widget v4.5.1 and @okta/okta-react v3.0.8.
```

Comment:
I think this format is more clear and doesn't leave the user wondering well - is there a newer version that needs to be tested with this guide now? Should I use these steps, then? I think the below format leaves less to question:

> This guide is for @okta/okta-signin-widget v4.5.1 and @okta/okta-react version 3.0.8.